pyUltroid/fns/helper.py

```diff
@@ -286,7 +288,7 @@ async def bash(cmd, run_code=0):
     err = stderr.decode().strip() or None
     out = stdout.decode().strip()
     if not run_code and err:
-        if match := re.match("\/bin\/sh: (.*): ?(\w+): not found", err):
+        if match := re.match("\\/bin\\/sh: (.*): ?(\\w+): not found", err):
```

Comment:
extra \ here

Reply:
when the lint auto fixed for some reason it didnt register 
if match := re.match("\/bin\/sh: (.*): ?(\w+): not found", err):
as if it had an r"\/bin\/sh: (.*): ?(\w+): not found"

only thing i can possible conclude